doc/md/fundamentals/2-types/9-mutable-arrays.md

```diff
@@ -94,14 +94,14 @@ for (i in arr.keys()) {
 
 ## Converting a mutable array to an immutable array
 
-You can convert a mutable array into an immutable array using `Array.freeze`, ensuring that the contents cannot be modified after conversion. Since mutable arrays are not [sharable](https://internetcomputer.org/docs/motoko/fundamentals/types/shared-types), freezing them is useful when passing data across [functions](https://internetcomputer.org/docs/motoko/fundamentals/types/functions) or [actors](https://internetcomputer.org/docs/motoko/fundamentals/actors-async) to ensure immutability.
+You can convert a mutable array into an immutable array using `Array.fromVarArray`, ensuring that the contents cannot be modified after conversion. Since mutable arrays are not [sharable](https://internetcomputer.org/docs/motoko/fundamentals/types/shared-types), freezing them is useful when passing data across [functions](https://internetcomputer.org/docs/motoko/fundamentals/types/functions) or [actors](https://internetcomputer.org/docs/motoko/fundamentals/actors-async) to ensure immutability.
 
 ```motoko no-repl
 import Array "mo:base/Array";
 
 let mutableArray : [var Nat] = [var 1, 2, 3];
 
-let immutableArray : [Nat] = Array.freeze<Nat>(mutableArray);
+let immutableArray : [Nat] = Array.fromVarArray<Nat>(mutableArray);
```

Comment:
Ditto

Reply:
The Tic-tac-toe exampels mentions VarArray.tabulate and VarArray.repeat which are new-base. Revert for now and add an html todo.